### ISSUE TITLE
low acc (below 90%) of training cifar10 from sketch using fbnet_a

### ISSUE THREAD

**Issue by lxiongh** [facebookresearch/mobile-vision#21]
my experiment's setting:
batch_size: 1000,
num_epoches=400
SGD, lr=0.1, momentum=0.9, weight_decay=1e-3
lr_schedule: torch.optim.lr_scheduler.MultiStepLR(optimizer, [100, 200, 300], gamma=0.1)

DataAug:
transforms.RandomCrop(32, padding=4),
transforms.RandomHorizontalFlip(),
transforms.RandomErasing(), 
transforms.ToTensor(),
